lib/views/activites/picture_select_activity_view.dart

```diff
@@ -84,8 +84,7 @@ class _PictureSelectActivityViewState extends State<PictureSelectActivityView> {
                     Column(
                       children: <Widget>[
                         Container(
-                          height: (MediaQuery.of(context).size.height / 100) *
```

Comment:
Please round

Reply:
I assume this is related to the above comment